qemu/qemu.go

```diff
@@ -43,6 +43,10 @@ type Machine struct {
 
 	// Acceleration are the machine acceleration options to be used by qemu.
 	Acceleration string
+
+	// ExtraOptions are extra options for the machine type
+	// For example gic-version=host and usb=off
+	ExtraOptions string
```

Comment:
You should call this `Options`, no need for the prefix `Extra`.

Reply:
ok,